### PR TITLE
Added unit tests for apps (Wasabee, pot2pump, Leaderboard, All-in-one-vault)

### PR DESCRIPTION
**Pull request by Shahzaibdev355**
Added unit tests for Wasabee, Pot2pump, Leaderboard, and All-in-one-vault apps.
Covered positive, negative, and edge cases for major functionalities.
Tests follow the required naming convention <file>.tests.ts and are placed under the /tests folder with the same folder structure.
Configured GitHub Actions to run tests on every commit.

### CODE REVIEW

**Comment by TBomberman** [comment]
Where are there 2 tests jobs? "test (18.x)" and "test (20.x)" ?

File: test/apps/all-in-one-vault/components/VaultDeposit.test.ts
Comment (TBomberman) [inline test/apps/all-in-one-vault/components/VaultDeposit.test.ts:1]: What file is this testing?

**Review by TBomberman — COMMENTED** [review]
I am unable to find the corresponding file under test such as BridgeSwap.ts.

File: apps/all-in-one-vault/jest.config.ts
Comment (TBomberman) [inline apps/all-in-one-vault/jest.config.ts:1]: There should be no jest files other than the apps folder and project root.

File: apps/all-in-one-vault/project.json
Comment (TBomberman) [inline apps/all-in-one-vault/project.json:1]: Mentioned not needed

File: apps/all-in-one-vault/hooks/useGetSupportTokenInfo.ts
Comment (TBomberman) [inline apps/all-in-one-vault/hooks/useGetSupportTokenInfo.ts:1]: The apps folder needs to be cleaned up from changes except the ones to fix testing errors.

File: apps/all-in-one-vault/test-exports.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// Barrel export file for testing purposes only
```
[the remark this answers is not in the document]
Comment (Shahzaibdev355) [reply apps/all-in-one-vault/test-exports.ts:1]: i will delete it
